### ISSUE TITLE
Additional information on user roles & rights?

### ISSUE THREAD

**Issue by markgrahamdunn** [contentsync/SketchContentSync#87]
Would it be possible to provide addtional comprehensive information related to user roles & rights in projects level & organization level?

Issue i had. 
I gave a colleague "Member" rights however he wasn't able to sync (pull & push) keys and values from his Sketch file. In his ContentSync overview dashboard he wan't able to access the Project i had given access to.

Only when i changed his role to Admin was he able to view the Project in his dashboard and sync (pull+push) via the Sketch plugin.

![image](https://user-images.githubusercontent.com/1683184/44518408-71f0a100-a6ca-11e8-9f9a-38789d0a50e1.png)

**Comment by dbrody** [comment]
@markgrahamdunn - Sure!

I've added a guide here:
[https://www.contentsync.io/docs/feature/user-permissions](https://www.contentsync.io/docs/feature/user-permissions)

I also noticed that yes, the permissions for running a command were incorrect - thats been updated so members should now be able to push/pull.

It looks like your colleague who is a member may still be pending as an invite? The yellow in your image  makes me think that. They wont be able to see the project until they accept the invite.
